packages/types/src/walletAdapter.ts

```diff
@@ -33,14 +33,14 @@ export type WalletAdapterData = {
 
   deeplink?: string;
 
-  connector: Connector;
+  createConnectorFn: CreateConnectorFn;
```

Comment:
I have named it this way because it is named same way in [wagmi typings](https://wagmi.sh/core/api/createConnector#parameters), for consistent naming.
I do not want to invent "our" version for the entity that already named by wagmi, it just appears to be confusing for me and unnecessary.

<img width="764" alt="image" src="https://github.com/lidofinance/reef-knot/assets/7289992/7a85c684-34cb-4bda-886c-df620c4a86a5">